src/main/scala/karazin/scala/users/group/week1/homework/Homework.scala

```diff
@@ -40,26 +40,79 @@ object Homework :
 
   object `Boolean Operators` :
 
-    def not(b: Boolean): Boolean = ???
+    def not(b: Boolean): Boolean = if (b) false else true
 
-    def and(left: Boolean, right: Boolean): Boolean = ???
+    def and(left: Boolean, right: Boolean): Boolean = if left then right else false

```

Comment:
As we discussed last lecture, sometimes it's not needed to calculate the right argument. Hint: use call-by-name strategy.
Don't forget to add tests for this case.